app/Console/MigrationsDropCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Console;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use function getenv;
+use function sprintf;
+
+class MigrationsDropCommand extends Command
```

Comment:
k čemu to je? myslel bych si, že mám uděta migration down a tím po sobě uklidit

Reply:
Tohle slouží pro dropnutí celé db pro testy. Nejde dělat down migraci, protože integrační testy "poškodí" db schéma. Tento command dočistí db včetně seznamu migrací a umožní v jednom běhu kontejneru ( volume je pezistentní), spustit všechny testy v libovolním pořadí. Down migrace je navíc zbytečně pomalá a už první migrace ji nemá korektně napsanou.

Reply:
ok